assignments/prototypes.js

```diff
@@ -92,19 +144,118 @@
     language: 'Elvish',
   });
 
+  const hero = new Humanoid({
+    createdAt: new Date(),
+    dimensions: {
+      length: 2,
+      width: 2,
+      height: 2,
+    },
+    hp: 9999,
+    name: 'Champion',
+    faction: 'Knight',
+    weapons: [
+      '2H Sword',
+      'Dart'
+    ],
+    language: 'Human',
+    attack: function(monster, dmg){
+      monster.hp -= dmg;
+      if (monster.hp <= 0){
+        return monster.destroy();
+      }
+      return `${this.name} attacks ${monster.name} with a ${this.weapons[0]} and does ${dmg} dmg.`;
+    },
+    throw: function(monster, dmg){
```

Comment:
I didn't even realize you weren't using the object.prototype.constructor syntax to make your hero and villain methods (like on line 90). I do think just making the method in the object is more applicable to the curriculum from here on out (which you did here).